packages/hub/src/components/exports/EditModelExports.tsx

```diff
@@ -75,15 +88,25 @@ export const EditModelExports: FC<Props> = ({ append, remove, items }) => {
                 {item.definition.username}/{item.definition.slug}
               </StyledDefinitionLink>
             </div>
-            <TrashIcon
-              className="text-slate-400 hover:text-slate-700 cursor-pointer"
-              onClick={() => remove(i)}
-            />
+            <StyledLink
+              href={modelForRelativeValuesExportRoute({
+                username: item.definition.username,
+                slug: modelSlug,
+                variableName: item.variableName,
+              })}
+            >
+              View
+            </StyledLink>
+            <Button onClick={() => remove(i)} size="small">
```

Comment:
It would be nice to make this red later, but that would require work on Button.

Reply:
I wanted to use a gray trash icon here, and even implemented it in the past PR but forgot to push before merging.

Red buttons next to each item would be too noisy, I think.

Reply:
I think it's fine if there are just a few elements. I often prefer being explicit (with words) for these things.  

On this page, the one action users can do is delete these exports.